updater/scripts/git-download.sh

```diff
@@ -14,23 +14,20 @@ function ghe_greater_equal () {
 }
 
 if ghe_greater_equal "2.11.0" ; then
-    # check the two most recent log files
-    CAT_LOG_FILE="zcat -f /var/log/github-audit.log{,.1*}"
     # The "github-audit.log" log file introduced in GHE 2.11.0 is only rolled
     # once a week. This was reported as a bug and is likely fixed in an
-    # upcoming version. In the meantime we grep for all log entries that have
-    # been written yesterday.
-    GREP_YESTERDAY="grep -F '$(date --date='yesterday' +'%b %d')'"
+    # upcoming version. In the meantime, we grep for all log entries in the two
+    # most recent log files (because the information from yesterday may or not
+    # be rotated already).
+    CAT_LOG_FILE="zcat -f /var/log/github-audit.log{,.1*} | grep -F '$(date --date='yesterday' +'%b %d')'"
 else
     # check yesterday’s log file
     CAT_LOG_FILE="zcat -f /var/log/github/audit.log.1*"
-    GREP_YESTERDAY="tee"
```

Comment:
Note that we can now get rid off this `tee` call when merging both `eval`s into one 😃.